docs: Improve readme (#337)
flybywiresim/discord-bot: README.md

![FlyByWire Simulations](https://raw.githubusercontent.com/flybywiresim/branding/master/tails-with-text/FBW-Color-Dark.svg#gh-light-mode-only)

[![Discord](https://img.shields.io/discord/738864299392630914.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.com/invite/flybywire)
[![Twitter](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=twitter&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://twitter.com/FlybywireSim)
[![YouTube](https://img.shields.io/badge/-FlyByWireSimulations-e84393?label=&logo=youtube&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://www.youtube.com/c/FlyByWire-Simulations)
[![Facebook](https://img.shields.io/badge/-FlyByWireSimulations-e84393?label=&logo=facebook&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://www.facebook.com/FlyByWireSimulations/)
[![Instagram](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=instagram&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://instagram.com/flybywiresim)

# FlyByWire Discord Bot

This is the repository for the FlyByWire Simulations Discord bot.

## Best Practices

> * Please use clear names for your PR's and commands

Reviewer: ```suggestion
* Please use clear names for your PR's and commands.
```

> * Check [here](https://github.com/flybywiresim/discord-bot/projects/1) to see a list of wanted commands that you may want to work on, or check the #discord-bot channel in the Discord Server

Reviewer: ```suggestion
* Check [here](https://github.com/flybywiresim/discord-bot/projects/1) what is being worked on, or check the #discord-bot channel in the Discord Server
* Check the #discord-bot channel for regular discussions about the repository.
```

Just a small edit I suggest

[![Twitter](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=twitter&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://twitter.com/FlybywireSim)
[![YouTube](https://img.shields.io/badge/-FlyByWireSimulations-e84393?label=&logo=youtube&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://www.youtube.com/c/FlyByWire-Simulations)
[![Facebook](https://img.shields.io/badge/-FlyByWireSimulations-e84393?label=&logo=facebook&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://www.facebook.com/FlyByWireSimulations/)
[![Instagram](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=instagram&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://instagram.com/flybywiresim)

# FlyByWire Discord bot
# FlyByWire Discord Bot

This is the repository for the FlyByWire Simulations Discord bot.

## Best Practices

* Please use clear names for your PR's and commands
* Check [here](https://github.com/flybywiresim/discord-bot/projects/1) to see a list of wanted commands that you may want to work on, or check the #discord-bot channel in the Discord Server

> * Once you have created your PR, send a message in the #discord-bot channel, or message BenW#8484, █▀█ █▄█ ▀█▀#2123 or oim#0001 on discord, and it will be added to the [projects page](https://github.com/flybywiresim/discord-bot/projects/1). This helps contributors see who is working on what

Reviewer: Not really.

> * Once you have created your PR, send a message in the #discord-bot channel, or message BenW#8484, █▀█ █▄█ ▀█▀#2123 or oim#0001 on discord, and it will be added to the [projects page](https://github.com/flybywiresim/discord-bot/projects/1). This helps contributors see who is working on what

Reviewer 2: > Not really.

Why not? It's fairly useful info to know

> * Once you have created your PR, send a message in the #discord-bot channel, or message BenW#8484, █▀█ █▄█ ▀█▀#2123 or oim#0001 on discord, and it will be added to the [projects page](https://github.com/flybywiresim/discord-bot/projects/1). This helps contributors see who is working on what

Reviewer: People do it anyways don't need to specify and is it really a "Best Practice"?

Don't really see the need to specify 🙂
